plugin-management-library/src/main/java/io/jenkins/tools/pluginmanager/impl/PluginManager.java

```diff
@@ -129,13 +130,35 @@ public PluginManager(Config cfg) {
         httpClient = null;
     }
 
+    private String getUserAgentInformation(){
+        String userAgentInformation="";
+        Properties properties = new Properties();
+            ClassLoader propertiesClassLoader = this.getClass().getClassLoader();
+            InputStream propertiesStream = propertiesClassLoader.getResourceAsStream("Version.properties");
```

Comment:
the resource is not closed property. Please use Java's try-with-resources.

Reply:
I will refactor this.